scalajslib/worker/1/src/mill/scalajslib/worker/ScalaJSWorkerImpl.scala

```diff
@@ -169,7 +171,8 @@ class ScalaJSWorkerImpl extends ScalaJSWorkerApi {
       esFeatures: ESFeatures,
       moduleSplitStyle: ModuleSplitStyle,
       outputPatterns: OutputPatterns,
-      minify: Boolean
+      minify: Boolean,
+      esModuleMap: Map[String, String] = Map[String, String]()
```

Comment:
Can you avoid the default argument here and pass `esModuleMap = Map.empty` at callsite?